### ISSUE TITLE
Overflow behavior on mobile

### ISSUE THREAD

**Issue by benharri** [bluesky-social/social-app#811]
Mobile layouts are not displaying PostMeta correctly

              Thank you @benharri for being so patient with me

For some reason this totally screws up the overflow behaviors on iOS

<img width="402" alt="CleanShot 2023-05-30 at 18 29 18@2x" src="https://github.com/bluesky-social/social-app/assets/1270099/67a7590d-3a15-452b-8ef6-12ca56263da8">

We have some challenges with this on android already, and might need to bundle this with some more robust approach to text layout in the PostMeta control. We either need to wrap our heads around react native's layout styles better (and we've been _trying_) or we need to get some manual sizing controls in there.

_Originally posted by @pfrazee in https://github.com/bluesky-social/social-app/issues/769#issuecomment-1569272996_

**Comment by ljstella** [comment]
This change also has some "odd" behaviors in quoteskeets on the web client: 

![Screenshot 2023-05-30 at 21 46 47](https://github.com/bluesky-social/social-app/assets/1413514/f7143575-e64b-4008-834f-ac885f90d3d3)

**Comment by loganrosen** [comment]
Another data point: Weird Al's handle _and_ name are getting completely replaced by an ellipsis on my Pixel 7 Pro:

![Screenshot_20230605-233135.png](https://github.com/bluesky-social/social-app/assets/512317/4158cbe8-ff1d-4b15-8a94-d5a0b6a06c5f)

**Comment by pfrazee** [comment]
Yeah I can't get react native's flex system to properly handle this. I'm leaning toward manual layout calculations

**Comment by benharri** [comment]
thoughts on using the 2 line format everywhere? can just skip a lot of the issues here

**Comment by haileyok** [comment]
Closing, this has been fixed it seems.
